src/components/button/button.md

```diff
@@ -53,6 +53,23 @@ Secondary, disabled:
       <Button variant="secondary" disabled modifier="warning" style={ style }>Click Me</Button>
       <Button variant="secondary" disabled modifier="success" style={ style }>Click Me</Button>
     </div>
+    
+Secondary, icon:
```

Comment:
No `primary` icons?

Reply:
I removed them as @There5a said we shouldn't use them. They're still available, just not in the documentation to avoid confusion